app/src/main/java/com/arygm/quickfix/model/search/SearchViewModel.kt

```diff
@@ -46,6 +50,10 @@ open class SearchViewModel(
     _searchQuery.value = query
   }
 
+  fun setSearchSubcategory(subcategory: Subcategory) { // Used for test purposes
```

Comment:
It's better to remove the comments here since you're using this function on the ExpandableCategoryItem